transport/sqs/src/main/kotlin/SqsConfig.kt

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2024 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.eclipse.apoapsis.ortserver.transport.sqs
+
+import aws.sdk.kotlin.runtime.auth.credentials.StaticCredentialsProvider
+import aws.sdk.kotlin.services.sqs.SqsClient
+import aws.smithy.kotlin.runtime.net.url.Url
+
+import org.eclipse.apoapsis.ortserver.config.ConfigManager
+import org.eclipse.apoapsis.ortserver.config.Path
+
+internal const val SERVER_URI_PROP_NAME = "serverUri"
+internal const val ACCESS_KEY_ID_PROP_NAME = "accessKeyId"
+internal const val SECRET_ACCESS_KEY_PROP_NAME = "secretAccessKey"
+internal const val QUEUE_NAME_PROP_NAME = "queueName"
+
+/**
+ * This class defines the configuration properties for an SQS transport. Specifically, this targets the implementation
+ * at Scaleway, see https://www.scaleway.com/en/developers/api/messaging-and-queuing/sqs-api/.
```

Comment:
The message of the last sentence is a bit unclear to me. Does it mean that for AWS a different configuration will be required? Or was it just not tested with AWS?

Reply:
Is was just not tested with AWS directly. I've clarified the sentence.